scalafix-tests/unit/src/test/scala/scalafix/tests/GitHubUrlRewriteSuite.scala

```diff
@@ -2,37 +2,42 @@ package scalafix
 package tests
 
 import scalafix.reflect.ScalafixCompilerDecoder.GitHubUrlRewrite
-
-import org.scalatest.{FunSuiteLike, Matchers}
 import metaconfig.Conf
+import org.scalatest.FunSuite
 
-class GitHubUrlRewriteSuite extends FunSuiteLike with Matchers {
-  test("it expands a GitHub shorthand with a default sha") {
-    Conf.Str("github:someorg/somerepo/1.2.3") match {
-      case GitHubUrlRewrite(url) =>
-        url.toString shouldBe "https://github.com/someorg/somerepo/blob/master/scalafix-rewrites/src/main/scala/somerepo/scalafix/Somerepo_1_2_3.scala"
+class GithubUrlRewriteSuite extends FunSuite {
+  def check(original: String, expected: String): Unit = {
```

Comment:
FYI @gabro I generally prefer FunSuite + assert. I also rarely name my tests since they tend to get off sync with the test implementation.